flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/Property.java

```diff
@@ -74,9 +74,12 @@ public <T> T valueAsType(TypeToken<T> typeToken) {
 
     public static final String VARIABLE_KEY = "variable";
     public static final String VARIABLE_NAME = "Variable Name";
+    public static final String VARIABLE_DOC = "Name of the variable";
     public static final String PARAMETER_VARIABLE_DOC = "Name of the parameter";
     public static final String IMPLICIT_VARIABLE_LABEL = "Name";
-    public static final String VARIABLE_DOC = "Name of the variable";
+
+    public static final String RESULT_NAME = "Result";
+    public static final String RESULT_DOC = "Name of the result variable";
```

Comment:
@hasithaa please review the new label + description used for the result variable of http client action calls